nucypher/blockchain/eth/sol/source/contracts/PolicyManager.sol

```diff
@@ -146,11 +165,52 @@ contract PolicyManager is Upgradeable {
     }
 
     /**
-    * @notice Set the minimum reward that the node will take
+    * @notice Set range for the minimum reward rate for all nodes
+    */
+    // TODO # 1501
+    // function setMinRewardRateRange(Range calldata _range) external onlyOwner {
+    function setMinRewardRateRange(uint256 _min, uint256 _default, uint256 _max) external onlyOwner {
+        require(_min <= _default && _default <= _max);
+        minRewardRateRange = Range(_min, _default, _max);
+        emit MinRewardRateRangeSet(msg.sender, _min, _default, _max);
+    }
+
+    /**
+    * @notice Set the minimum reward acceptable by node
+    * @dev Input value must be within `minRewardRateRange`
     */
     function setMinRewardRate(uint256 _minRewardRate) external {
-        NodeInfo storage node = nodes[msg.sender];
-        node.minRewardRate = _minRewardRate;
+        require(_minRewardRate >= minRewardRateRange.min &&
+            _minRewardRate <= minRewardRateRange.max,
+            "The min reward rate must be within permitted range");
```

Comment:
```suggestion
            "The minimum rate must fall within the universal fee range");
```
We need a clear and descriptive name for the price range that we use consistently all over the codebase, documentation and research – this will help understanding and future community discussions.  'Universal fee range' is the best I can think of right now

Reply:
Global/universal fee range doesn't show that this range for rate (per period per node) and not for price for policy moreover this range only for minimum node value and Alice can set rate higher than range.maximum